### ISSUE TITLE
pod failed with 0.25.0-rc

### ISSUE THREAD

**Issue by qq644531343** [facebook/react-native#7374]
1, My System is OS X 10.11.4 ,    ruby version is 2.3.1 (that dispay as 'ruby 2.3.1p112 (2016-04-26 revision 54768) [x86_64-darwin15]')  ， Xcode is 7.3， cocoapods 0.39.0

2，I cd to the Project's Root ，and execute :##  npm install react-native
     then the forder list as bellow:
`libodeMacBook-Pro:TBApp-React libo$ ls
Podfile			TBApp-OC		node_modules
Podfile.lock		TBApp-OC.xcodeproj
Pods			TBApp-OC.xcworkspace`
3，Edit the podfile   and  add :
  `pod 'React',            :path => 'node_modules/react-native',
                    :subspecs => [ 'Core', 'RCTImage', 'RCTNetwork', 'RCTText', 'RCTWebSocket',]`
4， rm Pods and Podfile.lock  ，then pod install ，
     but error occues:
 

Fetching podspec for `React` from `node_modules/react-native`
Fetching podspec for `VDLib` from `TBApp-OC/AppLib/libPod/VDLib`

[!] Invalid `React.podspec` file: No such file or directory - /Users/libo/Desktop/test/TBApp/TBApp-React/Pods/Local Podspecs/package.json. Updating CocoaPods might fix the issue.

 #  from /Users/libo/Desktop/test/TBApp/TBApp-React/Pods/Local Podspecs/React.podspec:3
 #  -------------------------------------------
 #  
 >  package = JSON.parse(File.read(File.join(__dir__, 'package.json')))
 #  
 #  -------------------------------------------

**Comment by rh389** [comment]
Could you post your whole Podfile? I've just tried reproducing this without success. `React.podspec` is normally executed from `node_modeules/react-native` and only a static `React.podspec.json` is created in `Pods/Local Podspecs`, but in your case it looks like the podspec is being moved/copied to an invalid location.

**Comment by qq644531343** [comment]
@rh389 

xcodeproj 'TBApp-OC.xcodeproj'

inhibit_all_warnings!

platform :ios, '7.0'

#######-------GitHub-------#######
pod 'AFNetworking'
pod 'JSONModel'
pod 'MJRefresh'
pod 'SDWebImage'
pod 'RegexKitLite'
pod 'ELCImagePickerController'   ##相册
pod 'DTCoreText'



#######------AppLib-------#######
#pod 'NetworkSDK',       :path => 'TBApp-OC/AppLib/libPod/ALINetworkSDK/NetworkSDK.podspec'
pod 'VDLib',           :path => 'TBApp-OC/AppLib/libPod/VDLib'
pod 'AppLib',           :path => 'TBApp-OC/AppLib'


pod 'React', :path => './node_modules/react-native', :subspecs => [
  'Core',
  'RCTImage',
  'RCTNetwork',
  'RCTText',
  'RCTWebSocket',
  # Add any other subspecs you want to use in your project
]




###### pod 常用引入方式
# pod 'Reachability', :path => 'a/b'
# pod 'Reachability'
# pod 'Reachability', '1.0'
# pod 'VDStruct', :svn => 'https://svn1...'
#
###### pod更新缓慢添加参数 --no-repo-update
# pod install --no-repo-update
#######

#target 'TBApp-OC' do
#    platform:ios, '7.0'
#    post_install do |installer|
#        puts 'Updating project to 64 Bit'
#        installer.project.targets.each do |target|
#            target.build_configurations.each do |config|
#                config.build_settings['ARCHS'] = "$(ARCHS_STANDARD)"
#                config.build_settings['ONLY_ACTIVE_ARCH'] = "NO"
#            end
#        end
#    end
#end

**Comment by taylesworth** [comment]
I have the same issue.

OS X 10.11.4
ruby 2.3.0p0 (2015-12-25 revision 53290) [x86_64-darwin15]
CocoaPods 1.0.0

> Fetching podspec for `React` from `./node_modules/react-native`
> 
> [!] Invalid `React.podspec` file: No such file or directory - /Users/tom.aylesworth/repo/ios/Pods/Local Podspecs/package.json.
> 
>  #  from /Users/tom.aylesworth/repo/ios/Pods/Local Podspecs/React.podspec:3
>  #  -------------------------------------------
>  #  
>  \>  package = JSON.parse(File.read(File.join(__dir__, 'package.json')))
>  #  
>  #  -------------------------------------------
>

**Comment by taylesworth** [comment]
Ok I managed to fix it by removing my `Podfile.lock` and the entire `Pods` directory. After doing that, `pod install` installed correctly.

I suspect this was caused by having previously done a `pod install` with CocoaPods 0.38, which copied the entire `React.podspec` to `Local Podspecs`. I then upgraded to CocoaPods 1.0.0 which tried to use the `Local Podspecs` version of `React.podspec` giving the error.

After blowing away the `Pods` directory, CocoaPods 1.0.0 `pod install` worked and, as @qq644531343 said, only placed a `React.podspec.json` in `Local Podspecs`.
